### PR TITLE
[RuboCop] Update to 0.33.0

### PR DESCRIPTION
**Pull request by segiddins**

### CODE REVIEW

File: lib/bundler/graph.rb [hunk 1]
```diff
@@ -135,7 +135,7 @@ def run
         @groups.each do |group|
           g.add_nodes(
             group,
-            { :style     => "filled",
+            { :style => "filled",
```
Comment (indirect) [inline lib/bundler/graph.rb]: this makes no sense to me... why is the first line not allowed to be aligned, but the rest are?

Comment (segiddins) [inline lib/bundler/graph.rb]: oooooh good question, didn't notice that one (autocorrect)

**Comment by segiddins** [comment]
@indirect we ought to get this merged so that `master` is back passing?

**Comment by andremedeiros** [comment]
Looks good to me. :shipit:

**Comment by indirect** [comment]
I was just waiting for some resolution on the weird "first line can't align but the other lines can" cop. I guess it expects the `{` to be on a line by itself.

**Comment by segiddins** [comment]
@indirect probably, but I didn't think it was worth keeping the build broken over

**Pull request merged**